Cargo.toml

```diff
@@ -18,14 +18,14 @@ bitcoin = { version = "0.29.2", features = ["rand"] }
 clap = { version = "4.4.6", features = ["derive"] }
 futures = "0.3.26"
 home = "0.5.5"
-lightning = "0.0.118"
+lightning = { git = "https://github.com/orbitalturtle/rust-lightning", branch = "signature-data-enum" }
```

Comment:
Can you open up an issue tracking our dependencies where we're waiting for merge for functionality? Just want to keep track of it on a higher level so that we don't loose track of things as they get merged.